spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -268,15 +273,20 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 			// and expand them to their long form if applicable
 			AppDefinition revisedDefinition = mergeAndExpandAppProperties(currentApp, metadataResource, appDeployTimeProperties);
 			AppDeploymentRequest request = new AppDeploymentRequest(revisedDefinition, appResource, deployerDeploymentProperties);
-
 			try {
+				String loggingString = String.format("Deploying the app '%s.%s'", currentApp.getStreamName(),
```

Comment:
getURI is more portable than getFilename and provides a bit more information for maven resources.